### PR TITLE
[BPK-1432] Add Flow typing to BpkPicker

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

File: native/packages/react-native-bpk-component-picker/src/common-types.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { type Node } from 'react';
+import PropTypes from 'prop-types';
+
+export type PickerValue = string | number;
```
Comment (k0nserv) [inline native/packages/react-native-bpk-component-picker/src/common-types.js]: Why do we support both string and number?

File: native/packages/react-native-bpk-component-picker/src/common-types.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+export type PickerMenu = {
```
Comment (k0nserv) [inline native/packages/react-native-bpk-component-picker/src/common-types.js]: This should be `PickerMenuProps` right?

File: native/packages/react-native-bpk-component-picker/src/common-types.js (hunk 1 above)
Comment (shaundon) [reply native/packages/react-native-bpk-component-picker/src/common-types.js]: We're matching what the underlying Picker component uses: https://github.com/facebook/react-native/blob/master/Libraries/Components/Picker/Picker.js#L53

(It says `any` but then right above it, says string or number, so I went with the stricter one).

File: native/packages/react-native-bpk-component-picker/src/common-types.js (hunk 2 above)
Comment (shaundon) [reply native/packages/react-native-bpk-component-picker/src/common-types.js]: Ah yeah, will fix.

Comment (shaundon) [reply native/packages/react-native-bpk-component-picker/src/common-types.js]: Done

**Pull request merged**